x-pack/plugins/rollup/public/crud_app/constants/validation.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+
+export const ILLEGAL_CHARACTERS = {
+  ROLLUP_INDEX: [ ...INDEX_ILLEGAL_CHARACTERS_VISIBLE, '*' ]
+};
```

Comment:
Minor nit, but can we just export `ILLEGAL_CHARACTERS_ROLLUP_INDEX`? I'm not sure if a wrapper object adds any value.

Reply:
good point, i'll update.

Reply:
fixed